vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/logparser/parser/TestRunProgressParser.kt

```diff
@@ -14,14 +13,14 @@ import kotlin.math.roundToInt
 
 class TestRunProgressParser(
     private val timer: Timer,
-    private val packageNameFormatter: PackageNameFormatter,
+    private val targetName: String,
 ) : TestEventProducer {
 
     val logger = MarathonLogging.logger(TestRunProgressParser::class.java.simpleName)
 
-    val TEST_CASE_STARTED = """Test Case '-\[([a-zA-Z0-9_.]+) ([a-zA-Z0-9_]+)]' started\.""".toRegex()
+    val TEST_CASE_STARTED = """Test Case '-\[([a-zA-Z0-9_.]+) ([a-zA-Z0-9_ ]+)]' started\.""".toRegex()
```

Comment:
added space, right?

Reply:
Added space to the test method group. Flutter allows spaces